007-openshift-yaml-health-probes/README.md

```diff
@@ -1,7 +0,0 @@
-# Quarkus - OpenShift extension
```

Comment:
Would keep this file and point towards the new module where the coverage was migrated